.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: python
+python: "2.7"
+before_install:
+ - sudo apt-get update -qq
+ - sudo apt-get install -qq python-apt python-pycurl
+install:
+  - pip install ansible==1.8.0
+script:
+  - ansible-galaxy install -r tests/requirements.yml
+  - ansible-playbook -i tests/inventory --syntax-check --list-tasks tests/test.yml -e "role_name=ansible-role-sublimetext3_packagecontrol" -e "hosts_group=hosts_group"
```

Comment:
Yep that is lazy copy pasting on my part. I originally half stole my ideas for travis code from https://github.com/jdauphant/ansible-role-nginx and https://github.com/RHInception/ansible-xml 

I was new to Travis and had a lot of the 'DAMN YOU TRAVIS!' commits. I also want to say when I ran an older version of Ansible it would throw an error if `role_name` !== the role's name. I tried it out on 1.8.0 and that doesn't seem to be the case anymore. 

The hosts_group is 100% uneccessary.

Thanks for catching that! I have that code in way too many of my ansible roles. I need to go an updated that. Even worse I submitted a pull request to someone with that code and he has been copy pasting it too. I'll be sure to let him know and credit you with for pointing out the mistake. 

Thanks,
Chris